retrieve_data.sh

```diff
@@ -1,22 +1,30 @@
 #!/bin/bash
 
+# Option to specify where to save the MERRA-2 files (MS)
+dpth='/Volumes/Samsung_T1/MTEST' #save in present directory
+# dpth='/PATH/TO/DIRECTORY' #e.g. if you want to put on an externa drive
+
 # This script will retrieve data by executing wget URLs. 
-cd surface
-rm MERRA2*
-rm tmp*
-wget -c -nc --load-cookies ~/.urs_cookies --save-cookies ~/.urs_cookies --auth-no-challenge=on --keep-session-cookies -i wget_surface.sh
+# cd surface
+# rm MERRA2*
+# rm tmp*
+# wget -P $dpth --load-cookies ~/.urs_cookies --save-cookies ~/.urs_cookies --auth-no-challenge=on --keep-session-cookies --content-disposition -i wget_surface.sh
+# cd ..
 
-cd ../radiation
+cd radiation
 rm MERRA2*
 rm tmp*
-wget -c -nc --load-cookies ~/.urs_cookies --save-cookies ~/.urs_cookies --auth-no-challenge=on --keep-session-cookies -i wget_radiation.sh
+wget -P $dpth --load-cookies ~/.urs_cookies --save-cookies ~/.urs_cookies --auth-no-challenge=on --keep-session-cookies --content-disposition -i wget_radiation.sh
+cd ..
 
-cd ../precip
-rm MERRA2*
-rm tmp*
-wget -c -nc --load-cookies ~/.urs_cookies --save-cookies ~/.urs_cookies --auth-no-challenge=on --keep-session-cookies -i wget_precip.sh
+# cd precip
+# rm MERRA2*
+# rm tmp*
+# wget -P $dpth --load-cookies ~/.urs_cookies --save-cookies ~/.urs_cookies --auth-no-challenge=on --keep-session-cookies --content-disposition -i wget_precip.sh
+# cd ..
 
-cd ../evap
-rm MERRA2*
-rm tmp*
-wget -c -nc --load-cookies ~/.urs_cookies --save-cookies ~/.urs_cookies --auth-no-challenge=on --keep-session-cookies -i wget_evap.sh
+# cd evap
+# rm MERRA2*
+# rm tmp*
+# wget -P $dpth --load-cookies ~/.urs_cookies --save-cookies ~/.urs_cookies --auth-no-challenge=on --keep-session-cookies --content-disposition -i wget_evap.sh
+# cd ..
```

Comment:
Here and elsewhere: Looks like this commit will remove evap, etc downloads. Can you add them back? Or am I missing something?

Reply:
Yes, again this was for my own use - I didn't want to download everything in one go, e.g. I just wanted to get the radiation fields.